src/main/java/edu/cornell/kfs/rass/batch/service/RassService.java

```diff
@@ -0,0 +1,15 @@
+package edu.cornell.kfs.rass.batch.service;
+
+public interface RassService {
+	
+	void readXML();
+	
+	boolean updateKFS();
+	
+	boolean updateAgencies();
```

Comment:
should updateAgencies, updateProposals and updateAwards just be protected functions on the implementations and not be part of the interface?  The job should just call updateKFS right?

Reply:
good point, will update